QualityControl/lib/services/ccdb/CcdbService.js

```diff
@@ -89,7 +89,10 @@ export class CcdbService {
       throw new Error(`Unable to connect to CCDB due to: ${error}`);
     }
     try {
-      const version = Object.values(serviceInfo[CCDB_MONITOR])[0][0]?.value ?? '-';
+      const monitorData = serviceInfo[CCDB_MONITOR] && serviceInfo[CCDB_MONITOR][this._hostname];
+      const version = Array.isArray(monitorData) && monitorData[0]?.value
```

Comment:
I think these 2 lines are not easy to read and can be improved. What do you think of:
```suggestion
      const monitorData = serviceInfo?.[CCDB_MONITOR]?.[this._hostname] ?? [];
      const version = monitorData[0]?.value ?? 'unknown version';
```